README.md

```diff
@@ -322,23 +322,27 @@ kubectl apply -f https://raw.githubusercontent.com/projectriff-demo/demo/master/
 
 ### Watch the orders stream
 
-Set up service account (skip if you already have this configured)
+Set up role bindings for dev-utils:
 
 ```
-kubectl create serviceaccount dev-utils --namespace default
-kubectl create rolebinding dev-utils --namespace default --clusterrole=view --serviceaccount=default:dev-utils
+kubectl create clusterrolebinding dev-util-stream --clusterrole=riff-streaming-readonly-role --serviceaccount=default:default
+kubectl create clusterrolebinding dev-util-core --clusterrole=riff-core-readonly-role --serviceaccount=default:default
```

Comment:
we don't use the core or knative features for dev-utils but I guess they won't hurt